extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/results/ExportResultsFrameProcessorFactory.java

```diff
@@ -107,7 +112,14 @@ public ProcessorsAndChannels<Object, Long> makeProcessors(
     );
 
     if (inputSliceReader.numReadableInputs(slice) == 0) {
-      return new ProcessorsAndChannels<>(ProcessorManagers.none(), OutputChannels.none());
+      return new ProcessorsAndChannels<>(
+          ProcessorManagers.of(Sequences.<ExportResultsFrameProcessor>empty())
+                           .withAccumulation(new ArrayList<String>(), (acc, file) -> {
+                             ((ArrayList<String>) acc).add((String) file);
+                             return acc;
+                           }),
```

Comment:
Seems confusing, do we need this?

Reply:
We need the withAccumulation call so that the types match, but I have changed it to be a noop instead, more in line with other places. https://github.com/apache/druid/blob/9c7d7fc77748a6b8534f1ec3c36e226c63f8cfe7/extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/processor/SegmentGeneratorFrameProcessorFactory.java#L132